data/lang/equipment-core/en.json

```diff
@@ -151,6 +155,10 @@
     "description": "",
     "message": "An electronic countermeasure missile defence system, capable of destroying some homing missiles."
   },
+  "ECM_BASIC_HOVER_MESSAGE": {
+    "description": "",
+    "message": "Electronic Countermeassure"
```

Comment:
Properly rendered in English, this would be "Electronic Countermeasure System" or "Electronic Countermeasures" (note the single 's' in countermeasure). As this is a hover message, I'd much prefer "Activate Electronic Countermeasures" as that explains the action rather than the system. This also allows the string to be used when we rework the missile / countermeasures system interface sometime soon:tm:.